templates/guides/vault-scaling.md.tmpl

```diff
@@ -1,5 +1,5 @@
 ---
-subcategory: ""
+subcategory: "HCP Vault"
```

Comment:
Does this put the guides under the `HCP Vault` subecategory? I think we'd still want the guides to be top level

Reply:
This would put this specific guide under that category, in the same way that resources and data sources work. I have left guides that are cross-product or non-product-specific at the root level, but I figure product-specific guides should go alongside the product's resources/data sources

Reply:
Gotcha. That seems reasonable.